src/yodapa/plugins/asciiator.py

```diff
@@ -0,0 +1,82 @@
+from PIL import Image
+import os
+import typer
+
+app = typer.Typer(help="""
+    Asciiator plugin. Convert an image to ASCII art.
+
+    Example:
+
+        $ yoda asciiator process "{path to image file}" -s // save output locally
+        $ yoda asciiator process "{path to image file}" -t // display output on terminal
+        $ yoda asciiator process "{path to image file}" -s -t // both the actions
+      
+    """)
+
+ASCII_CHARS = ["#", "?", "%", ".", "S", "+", ".", "*", ":", ",", "@"]
+
+def handle_image_conversion(image_filepath):
+    image = None
+    try:
+        image = Image.open(image_filepath)
+    except Exception as e:
+        print(f"Unable to open image file {image_filepath}.")
+        print(e)
```

Comment:
use print from the `rich` package. For that you can simply add `from rich import print` at the top and you won't have to change anything else in this file

Reply:
alright